### PR TITLE
XW-2380 | Translations for Create Page dialog

### PR DESCRIPTION
**Pull request by rybmat**
https://wikia-inc.atlassian.net/browse/XW-2380

//cc @Wikia/x-wing

### CODE REVIEW

**Review by hakubo — APPROVED** [review]

File: extensions/wikia/CreatePage/CreatePage.i18n.php [hunk 1]
```diff
@@ -1,9 +1,8 @@
 <?php
-/** Internationalization file for /extensions/wikia/CreatePage/CreatePage extension. */
-$messages = [];
+$messages = array();
 
-$messages['en'] = [
-	'createpage-desc' => 'Allows to create a new page using Wikia\'s WYSIWYG editor',
+$messages['en'] = array(
```
Comment (rogatty) [inline extensions/wikia/CreatePage/CreatePage.i18n.php]: This is an old syntax. Why do we change it?

File: extensions/wikia/CreatePage/CreatePage.i18n.php [hunk 2]
```diff
@@ -343,44 +342,36 @@
...
+$messages['de'] = array(
+	'createpage-desc' => 'Erlaubt das Erstellen von neuen Seiten mit Fandoms WYSIWYG-Editor',
 	'createpage-sp-title' => 'Neue Seite anlegen',
 	'createpage_title' => 'Neue Seite anlegen',
-	'createpage_title_caption' => 'Seitentitel:',
+	'createpage_title_caption' => 'Titel:',
 	'createpage_enter_text' => 'Text:',
-	'createpage_empty_title_error' => 'Titel erforderlich',
-	'createpage_empty_article_body_error' => 'Artikeltext erforderlich',
+	'createpage_empty_title_error' => 'Please write a title for your new article!',
```
Comment (rogatty) [inline extensions/wikia/CreatePage/CreatePage.i18n.php]: It was translated before, why do we make it English? There are multiple cases of this.

File: extensions/wikia/CreatePage/CreatePage.i18n.php (hunk 1 above)
Comment (rogatty) [inline extensions/wikia/CreatePage/CreatePage.i18n.php]: This is an old syntax. Why do we change it?

**Review by rogatty — CHANGES_REQUESTED** [review]

File: extensions/wikia/CreatePage/CreatePage.i18n.php (hunk 1 above)
Comment (rybmat) [reply extensions/wikia/CreatePage/CreatePage.i18n.php]: so should we change those manually or fix i18n-tools?

File: extensions/wikia/CreatePage/CreatePage.i18n.php (hunk 2 above)
Comment (rybmat) [reply extensions/wikia/CreatePage/CreatePage.i18n.php]: this one is not translated in crowdin. It looks like somebody uploaded new keys without pulling translations before...

File: extensions/wikia/CreatePage/CreatePage.i18n.php [hunk 3]
```diff
@@ -346,32 +345,32 @@
 ];
 
 $messages['de'] = [
-	'createpage-desc' => 'Erlaubt das Erstellen von neuen Seiten mit Wikias WYSIWYG-Editor',
+	'createpage-desc' => 'Erlaubt das Erstellen von neuen Seiten mit Fandoms WYSIWYG-Editor',
 	'createpage-sp-title' => 'Neue Seite anlegen',
 	'createpage_title' => 'Neue Seite anlegen',
-	'createpage_title_caption' => 'Seitentitel:',
+	'createpage_title_caption' => 'Titel:',
```
Comment (rogatty) [inline extensions/wikia/CreatePage/CreatePage.i18n.php:351]: This one isn't on the list (https://wikia-inc.atlassian.net/browse/LOC-15699).

File: extensions/wikia/CreatePage/CreatePage.i18n.php [hunk 4]
```diff
@@ -346,32 +345,32 @@
...
+	'createpage-error-invalid-title' => "Dieser Titel ist ungültig. Folgende Zeichen dürfen nicht verwendet werden: # < > [ ] | { } Außerdem kann der Titel nicht mit 'Spezial:' beginnen.",
+	'createpage-error-article-exists' => 'Es existiert bereits eine Seite über <a href="$1">$2</a>. Bitte bearbeite diese oder wähle einen anderen Titel aus.',
+	'createpage-error-article-spam' => 'Dieser Titel klingt unpassend. Verwende bitte einen anderen Titel.',
+	'createpage-error-article-blocked' => 'Es ist etwas schief gelaufen. Wir arbeiten daran, das Problem zu beheben. Bitte probiere zu einem späteren Zeitpunkt noch einmal, deine neue Seite hinzuzufügen. Sollte das Problem weiterhin bestehen, nimm bitte Kontakt zu deinem Admin auf.',
 	'tog-createpagedefaultblank' => 'Wähle „Leere Seite“ als Standard bei Erstellung neuer Seiten',
-	'tog-createpagepopupdisabled' => 'Seite erstellen-Hinweis deaktivieren (Nicht empfohlen)',
-	'newpagelayout' => '[[Datei:Platzhalter|right|300px]]
+	'tog-createpagepopupdisabled' => 'Den Hinweis „Seite erstellen” deaktivieren (Nicht empfohlen)',
+	'newpagelayout' => '[[File:Placeholder|right|300px]]
```
Comment (rogatty) [inline extensions/wikia/CreatePage/CreatePage.i18n.php:373]: These two aren't on the list.

File: extensions/wikia/CreatePage/CreatePage.i18n.php [hunk 5]
```diff
@@ -394,6 +393,7 @@
 	'createpage-ve-body' => 'Der Artikel <b>$1</b> existiert in dieser Community nicht. Du kannst uns helfen, indem du ein paar Sätze hinzufügst.',
 	'createpage-button-proceed' => 'Neue Seite',
 	'createpage-button-cancel' => 'Abbrechen',
+	'createpage-dialog-redlinks-list-header' => 'Or click on one of the titles below and start writing!',
```
Comment (rogatty) [inline extensions/wikia/CreatePage/CreatePage.i18n.php:396]: This one isn't on the list.

File: extensions/wikia/CreatePage/CreatePage.i18n.php [hunk 6]
```diff
@@ -455,30 +455,30 @@
 ];
 
 $messages['es'] = [
-	'createpage-desc' => 'Permite crear una nueva página usando editor WYSIWYG de Wikia',
-	'createpage-sp-title' => 'Crea un nuevo artículo',
-	'createpage_title' => 'Crear un nuevo artículo',
+	'createpage-desc' => 'Permite crear una nueva página usando editor WYSIWYG de Fandom',
+	'createpage-sp-title' => 'Crea una nueva página',
+	'createpage_title' => 'Crea una nueva página',
```
Comment (rogatty) [inline extensions/wikia/CreatePage/CreatePage.i18n.php:460]: These two aren't on the list.

**Review by rogatty — CHANGES_REQUESTED** [review]

**Pull request closed without merging**